introduction/prototyping_gitbook.md

```diff
@@ -0,0 +1,17 @@
+# Prototyping gitbook
```

Comment:
this file won't show in the gitbook as it's not referenced in the summary.md
is that normal ?